src/Serializer/AbstractSerializer.php

```diff
@@ -241,7 +241,7 @@ protected function serializeValue($value)
         }
 
         if (\is_object($value)) {
-            return 'Object ' . \get_class($value);
+            return 'Object ' . \get_class($value) . (method_exists($value, 'getId') ? ' (#' . $value->getId() . ')'  : '');
```

Comment:
Using `property_exists` on `id` would be a bit more generic IMO.
Having some sort of test case that covers this would be appreciated as well, so we do not introduce regressions down the line.

Reply:
Good suggestion! I think it would be good to keep the `$object->getId()` fallback, since this a common way to expose an object's ID (ie. when using Doctrine entities)